server/tests/ballot_comparison/test_cvrs.py

```diff
@@ -646,7 +646,7 @@ def test_invalid_cvrs(
 CvrNumber,TabulatorNum,BatchId,RecordId,ImprintedId,CountingGroup,PrecinctPortion,BallotType,REP,DEM
 ,TABULATOR1,BATCH1,1,1-1-1,Election Day,12345,COUNTY,0,1
 """,
-            "CvrNumber is required. Missing CvrNumber in row 1.",
+            "Missing required column CvrNumber in row 1.",
```

Comment:
This is maybe an unnecessary nitpick, but it'd be great if we could demarcate that the column name must match that string a little better. Like, put it in quotes or something: `"Missing required column \"CvrNumber\" in row 1."`

Reply:
Not an unreasonable idea. Currently, this styling is consistent with the error messages from our other csv_parse module, so we'd probably want to change both of them if we want to make that change. I don't feel passionately about changing it, but if you do feel free! (Maybe in a separate PR)

Reply:
Heh, that sounds like an after-Christmas change to me. I should probably get @mcchilders and @ginvdr's suggestions.

Reply:
I literally finished the Troubleshooting guide and updating associated Front message templates this morning...  so I'm inclined to vote no for completely selfish reasons.  No strong feelings otherwise!